src/domain/enrolment/eventInfo/EventInfo.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { ReactElement } from 'react';
+
+import { EventFieldsFragment } from '../../../generated/graphql';
+import useLocale from '../../../hooks/useLocale';
+import getLocalisedString from '../../../utils/getLocalisedString';
+import styles from './eventInfo.module.scss';
+
+interface Props {
+  event: EventFieldsFragment;
+}
+
+const EventInfo = ({ event }: Props): ReactElement => {
```

Comment:
Would React.FC be more convenient in component definitions?:
const EventInfo: React.FC<Props> = ({ event }) => { ?

Reply:
Fixed